.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [ 1.15.x ]
+        go-version: [ 1.15.x, 1.16.x]
```

Comment:
Should I remove `1.15.x` here?

Reply:
I'm leaving it and updating the CONTRIBUTION.md to reflect 1.15 as the minimal version of Go.